frontend/src/pages/requests/StatusToolTip.tsx

```diff
@@ -0,0 +1,59 @@
+import { ITooltipParams } from "ag-grid-community";
+import { AgGridReact } from "ag-grid-react";
+import { useState } from "react";
+
+export const StatusTooltip = (props: ITooltipParams) => {
+  const data = props.api!.getDisplayedRowAtIndex(props.rowIndex!)!.data;
+
+  const [columnDefs] = useState([
+    { field: "fieldName", headerName: "Field Name" },
+    {
+      field: "report",
+      headerName: "Report",
+      wrapText: true,
+      autoHeight: true,
+      flex: 2,
+    },
+  ]);
```

Comment:
Looks like `columnDefs`'s value is set only once here, so I don't think we need to save it as a state via `useState()`.